### ISSUE TITLE
num_epochs ignored?

### ISSUE THREAD

**Issue by rthomason** [dennybritz/cnn-text-classification-tf#4]
I see num_epochs defaults to 200, but when I run it locally ( ./train.py ) it runs indefinitely.

For example, I stopped this run after 11000.

Parameters:
ALLOW_SOFT_PLACEMENT=True
BATCH_SIZE=64
CHECKPOINT_EVERY=100
DROPOUT_KEEP_PROB=0.5
EMBEDDING_DIM=128
EVALUATE_EVERY=100
FILTER_SIZES=3,4,5
L2_REG_LAMBDA=0.0
LOG_DEVICE_PLACEMENT=False
NUM_EPOCHS=200
NUM_FILTERS=128

...

Evaluation:
2016-01-08T19:28:26.785109: step 11000, loss 0.613605, acc 0.715

Saved model checkpoint to <my_path>/cnn-text-classification-tf/runs/1452309199/checkpoints/model-11000

**Comment by dennybritz** [comment]
Hi,

Epoch is not the same as step. An epoch is a complete iteration over the data set, but each step is the processing of one batch. So, 100 epochs = `DATA_SIZE/BATCH_SIZE * 100` steps.

**Comment by rthomason** [comment]
Thanks for clarifying!
